rust-src/concordium_base/src/transactions.rs

```diff
@@ -2226,6 +2274,106 @@ pub mod construct {
         )
     }
 
+    /// Construct a tokens transfer transaction.
```

Comment:
_Tokens_ is overloaded in this space, and even in this crate, so maybe explicitly mention protocol level tokens